frontend/public/components/catalog-items.jsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import * as PropTypes from 'prop-types';
+
+import {CatalogTileView} from 'patternfly-react-extensions/dist/esm/components/CatalogTileView';
+import {CatalogTile} from 'patternfly-react-extensions/dist/esm/components/CatalogTile';
+import {normalizeIconClass} from './catalog-item-icon';
+
+const CatalogItem = ({item}) => {
+  return <CatalogTile
```

Comment:
We should be handling tile clicks since we have pages to navigate to for each of these. Then we have a working catalog!

Reply:
yes, working on this now :-)